### PR TITLE
Add role to Directory Sync User object

### PR DESCRIPTION
**Pull request by ameesha**
## Description
Optional property on the directory sync user object.

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

File: workos/__about__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 __package_url__ = "https://github.com/workos-inc/workos-python"
 
-__version__ = "4.7.0"
+__version__ = "4.8.0"
```
Comment (mattgd) [inline workos/__about__.py:15]: Could we do this in a separate PR for any other changes that may be included in the release?

Comment (ameesha) [reply workos/__about__.py:15]: are you an owner for the python sdk? i do it in 1 so that it doesn't require 2 owners checks (been blocked by this before)
but i'll grab whatever changes that were merged before this when i cut a release

Comment (mattgd) [reply workos/__about__.py:15]: Yeah I'm a _Pythonista_, so can definitely approve the bump whenever. I suppose it's nice to have the clear history of when the version was bumped/what it included.

image

Comment (ameesha) [reply workos/__about__.py]: great, just undone

**Review by mattgd — APPROVED** [review]
